### PR TITLE
adapt to conu 0.4.0

### LINKED ISSUE #133: Adapt to conu 0.4.0
In conu 0.4.0 we are planning to introduce a breaking change in .get_metadata method. We should make sure that colin works with the new changes.

### PR DESCRIPTION
**Pull request by TomasTomecek**
Fixes #133

### CODE REVIEW

**Comment by TomasTomecek** [comment]
Fixed codacy issues, should be pretty now.

Gotta say I spent 5-10 minutes battling local test failures due to:
1. ruleset being loaded from ~/.local/...
2. outdated labels image

File: colin/core/checks/envs.py [hunk 1]
```diff
@@ -16,6 +16,7 @@
 
 import re
 
+from colin.core.target import inspect_object
```
Comment (lachmanfrantisek) [inline colin/core/checks/envs.py]: Only small nit: can we use relative imports for all? => `s/colin.core.target/..target`

**Comment by lachmanfrantisek** [comment]
>  ruleset being loaded from ~/.local/...

We should finally take a look into the ruleset loading.

> outdated labels image

Sorry, I haven't mentioned that in the previous PR.

**Comment by TomasTomecek** [comment]
> Sorry, I haven't mentioned that in the previous PR.

No worries.

Yup, ruleset loading would be a nice goal for next week.

Updated & fixed.

File: colin/core/checks/envs.py (hunk 1 above)
Comment (TomasTomecek) [reply colin/core/checks/envs.py]: My pycharm uses absolute paths, not sure why. I left it there on purpose and wondered whether you'll notice it :P

**Comment by TomasTomecek** [comment]
CI failure is 503 from fedora mirror servers -_-

**Pull request merged**